examples/custom-resources/grpc-upstreams/README.md

```diff
@@ -5,32 +5,65 @@ grpc** field to an upstream. The protocol defaults to http if left unset.
 
 ## Prerequisites
 
-- HTTP/2 must be enabled. See `http2` ConfigMap key in the
+1. HTTP/2 must be enabled. See `http2` ConfigMap key in the
   [ConfigMap](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/configmap-resource/#listeners)
 
-- VirtualServer and VirtualServerRoute resources for gRPC applications must include TLS termination.
-
-## Example
-
-```yaml
-apiVersion: k8s.nginx.org/v1
-kind: VirtualServer
-metadata:
-  name: grpc-vs
-spec:
-  host: grpc.example.com
-  tls:
-    secret: grpc-secret
-  upstreams:
-  - name: grpc1
-    service: grpc-svc
-    port: 50051
-    type: grpc
-  routes:
-  - path: /helloworld.Greeter
-    action:
-      pass: grpc1
+2. VirtualServer and VirtualServerRoute resources for gRPC applications must include TLS termination.
```

Comment:
```suggestion
2. Configure TLS termination for VirtualServer and VirtualServerRoute resources.
```

It's a requirement and in the context of gRPC applications, so mentioning it is redundant.